packages/x-components/src/x-modules/search-box/components/search-input.vue

```diff
@@ -319,14 +319,14 @@ the message “enter” appears.
 
 _Type any term in the input field to try it out!_
 
-```vue
+```vue live
 <template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work